cms/standard_pages/models.py

```diff
@@ -38,3 +40,76 @@ class InformationPage(BasePage):  # type: ignore[django-manager-missing]
         index.SearchField("summary"),
         index.SearchField("content"),
     ]
+
+
+class IndexPage(BasePage):  # type: ignore[django-manager-missing]
+    template = "templates/pages/index_page.html"
+
+    parent_page_types: ClassVar[list[str]] = ["home.HomePage", "IndexPage", "InformationPage"]
+    subpage_types: ClassVar[list[str]] = ["IndexPage", "InformationPage"]
+
+    description = models.TextField(blank=True)
+    featured_pages = StreamField([("featured_page", RelatedContentBlock())], blank=True)
+    content = RichTextField(features=settings.RICH_TEXT_BASIC, blank=True)
+    related_links = StreamField([("related_link", RelatedContentBlock())], blank=True)
+
+    content_panels: ClassVar[list["Panel"]] = [
+        *BasePage.content_panels,
+        FieldPanel("description"),
+        FieldPanel("featured_pages"),
+        FieldPanel("content"),
+        FieldPanel("related_links"),
+    ]
+
+    search_fields: ClassVar[list[index.SearchField | index.AutocompleteField]] = [
+        *BasePage.search_fields,
+        index.SearchField("description"),
+    ]
+
+    def get_formatted_featured_pages(self) -> list[dict[str, str]]:
+        formatted_featured_pages = []
+
+        if featured_pages := self.featured_pages:
+            for featured_page in featured_pages:
+                formatted_featured_pages.append(
+                    {
+                        "featured": "true",
+                        "title": {"text": featured_page.value.link["text"], "url": featured_page.value.link["url"]},
+                        "description": featured_page.value["description"],
+                    }
+                )
+
+        elif child_pages := self.get_children().live().public():
+            for child_page in child_pages:
+                formatted_featured_pages.append(
+                    {
+                        "featured": "true",
+                        "title": {"text": child_page.title, "url": child_page.url},
+                    }
+                )
```

Comment:
```suggestion
        else:
            for child_page in self.get_children().live().public().specific().defer_streamfields:
                formatted_featured_pages.append(
                    {
                        "featured": "true",
                        "title": {"text": child_page.title, "url": child_page.get_url(request=request)},
                        "description" child_page.listing_summary or child_page.summary
                    }
                )
```

this simplifies things a bit, and fetches specific instances, with deferred StreamField evaluation https://docs.wagtail.org/en/v6.3.2/reference/pages/queryset_reference.html#wagtail.query.PageQuerySet.defer_streamfields.

it also adds the subpage description